src/components/molecules/MessageFormatting/formats/LineBreaksAndParagraphs.tsx

```diff
@@ -15,7 +15,7 @@ export const LineBreaksAndParagraphs = () => (
       <code>
         {lineBreaksAndParagraphs.codeBlockText.map(({ id, item }) => (
           <span className={styles.inlineText} key={id}>
-            {item}
+            {item}&nbsp;&nbsp;
```

Comment:
Is this for layout purposes, because that's what padding is for.  Slight concern that if we have non-breaking spaces in our examples then they will end up in the content if people copy and paste which might stop them layout out correctly (may not be a real problem, depends on if copy-paste converts them to regular spaces).

Reply:
I confirmed with UCD the spaces are not for padding purposes. But exactly the reason you outlined. When users copy the example it will add the spaces and get the layout as expected.

Reply:
Did a little test and it looks like these are copied as standard space chars so should be fine.